org.scala-refactoring.library/src/main/scala/scala/tools/refactoring/common/tracing.scala

```diff
@@ -21,7 +26,32 @@ trait Tracing {
   def trace(msg: => String)
 }
 
-trait ConsoleTracing extends Tracing {
+object DebugTracing {
+  private val debugStream = {
+    Option(System.getProperty("scala.refactoring.traceFile")).flatMap { fn =>
```

Comment:
Since it's just for developers (and you need to re-compile the library anyway if you want to use it), I think we can just leave it in the string.